### ISSUE TITLE
meson/ninja scipy install not producing egg / dist info 

### ISSUE THREAD

**Issue by branfosj** [easybuilders/easybuild-easyblocks#2901]
And that means it does not appear in the result of `pip list` and causes anything requiring `scipy` to fail the sanity check.

**Comment by boegel** [comment]
We should have noticed this, so perhaps we should implement support for running `pip list | grep ext_name` as a part of the sanity check while we're at it?
```python
check_pip_list = True
```

@jfgrimm Any idea if we overlooked something, or whether `scipy` does something extra (that we don't) so `pip` is aware of it being installed?

**Comment by jfgrimm** [comment]
furthermore:
> It looks like there is a bug in SciPy-bundle/2023.02-gfbf-2022b: It does not install the include files for scipy and numpy, so other packages that compiles code building on top cannot install.  For example, I tried to make an easyconfig for scikit-learn, but that cannot build due to missing scipy header files.

**Comment by schiotz** [comment]
I was wrong when I reported that, the include files are where they should be (not where I expected them), and my trouble are solely due to pip claiming that scipy is not installed

**Comment by boegel** [comment]
It looks like we just need to copy the `PKG-INFO` that's included in the `scipy` source tarball (cfr. changes made in https://github.com/msys2/MINGW-packages/pull/12405):

```console
$ module load SciPy-bundle/2023.02-gfbf-2022b
$ pip list | grep scipy

[notice] A new release of pip available: 22.3.1 -> 23.0.1
[notice] To update, run: pip install --upgrade pip
```

```console
$ cp /tmp/$USER/scipy-1.9.3/PKG-INFO $EBROOTSCIPYMINBUNDLE/lib/python3.10/site-packages/scipy-1.9.3.egg-info
$ pip list | grep scipy
scipy                             1.9.3

[notice] A new release of pip available: 22.3.1 -> 23.0.1
[notice] To update, run: pip install --upgrade pip
```
